### PR TITLE
feat(wip): monorepo structure

### PR DESCRIPTION
**Pull request by SarjuHansaliya**
## Summary
This PR moves entire component structure to monorepo

## Checklist
- [x] core package
- [x] swap package
- [x] wallet-connectors package
- [x] wallet-modal package
- [x] SAR package
- [x] portfolio package
- [x] bridge-swap package
- [x] elixir package
- [x] pools package
- [x] governance package
- [x] airdrop package
- [x] shared package ( ongoing )
- [x] state-hooks package ( ongoing )
- [x] set up an example app that uses monorepo packages to make sure everything is working after migration
- [ ] set up automated changelog generation using changeset
- [ ] set up CI/CD actions to publish version. This should only go to the Honeycomb npm organisation. No need to duplicate it and publish to the historical Pangolin
- [ ] set up size-limit on each PR to make sure all packages are tree-shakable
- [ ] update readme, contribution guideline, github actions readme to include all essential information
- [ ] make sure storybook runs properly in each package
- [x] add/update ReadMe file in each package
- [ ] After everything is done, analyze all package json to make sure we have only necessary dependency
- [x] move all packages to `@honeycomb-finance` org. This should be dealt with above as part of the CI/CD process. 
- [ ] Integrate honeycomb packages to interface
- [ ] (optional) move common dependency to monorepo root 
- [ ] (optional) move rollup configuration to common package
- [ ] Resolve all SonarCloud issues

This PR is carry forwarded to https://github.com/Honeycomb-finance/components/pull/5

### CODE REVIEW

File: monorepo/README.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# Turborepo starter with Rollup
```
Comment (HariSeldon23) [inline monorepo/README.md:1]: @SarjuHansaliya can we include documentation about publishing to npm

Comment (SarjuHansaliya) [reply monorepo/README.md:1]: yes @HariSeldon23 , still working on all those things

**Comment by HariSeldon23** [comment]
@SarjuHansaliya could we create a checklist in this PR with outstanding issues before we can merge?

**Comment by SarjuHansaliya** [comment]
@HariSeldon23 I have added checklist to PR description

**Comment by SarjuHansaliya** [comment]
Closing this PR. This PR is superseded by https://github.com/Honeycomb-finance/components/pull/5

**Pull request closed without merging**